frontend/src/pages/Signup.js

```diff
@@ -16,11 +16,11 @@ const Signup = () => {
       <Title content="Join us to start spinning" />
 
       <div className="Signup-option-1">
-        <SocialMediaButton name="github" username="subvisual" />
-        <SocialMediaButton name="linkedin" username="wearesubvisual" />
-        <SocialMediaButton name="behance" username="subvisual" />
-        {/* <SocialMediaButton name="facebook" username="subvisual.co" />
-        <SocialMediaButton name="instagram" username="wearesubvisual" /> */}
+        <SocialMediaLink name="github" username="subvisual" />
+        <SocialMediaLink name="linkedin" username="wearesubvisual" />
+        <SocialMediaLink name="behance" username="subvisual" />
+        {/* <SocialMediaLink name="facebook" username="subvisual.co" />
+        <SocialMediaLink name="instagram" username="wearesubvisual" /> */}
       </div>
```

Comment:
You may want to think about who should store the data object. This page or `SocialMediaLink`